dbt/include/bigquery/macros/adapters.sql

```diff
@@ -222,3 +222,11 @@ having count(*) > 1
   {% do adapter.upload_file(local_file_path, database, table_schema, table_name, kwargs=kwargs) %}
 
 {% endmacro %}
+
+{% macro bigquery__upload_run_results(local_file_path, database, table_schema, table_name) %}
+
+  {{ log("kwargs: " ~ kwargs) }}
```

Comment:
Let's move the log message up into the `upload_run_results` (Python method), where it's slightly easier to construct a more descriptive message, and pass into the `logger` (= BigQuery `AdapterLogger`)

Reply:
Done! 32afa0eb243cd3f529a8c73f1fee066e152c0215

I've chosen "debug" rather than "info" however the print out still contains a large JSON string for the schema which can be offputting. Would be nice if there was an elegant way to only print this when the "-d" flag is passed.